modules/fastqc/functions.nf

```diff
@@ -3,10 +3,10 @@
 //
 
 //
-// Extract name of software tool from process name using $task.process
+// Extract name of module from process name using $task.process
 //
-def getSoftwareName(task_process) {
```

Comment:
We will still need `getSoftwareName` too because there is an additional `tokenize` to get the software name? I think it's much cleaner to have functions for this sort of thing rather than a bunch of operations in the module itself?

Reply:
Reverted changes for this in the other pull. 
Better to keep the function.